### PR TITLE
IN-5582: Added special tags

### PR DESCRIPTION
**Pull request by adnanrahic**
Added the basic structure for Special Tags.
Todo: Wording.

### CODE REVIEW

File: docs/logs/special-fields.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 title: Special Logs Fields
 description: Sematext's logging management app looks for special fields in logs, namely host, source, facility, severity, syslog-tag, tags, message, and @timestamp
 
+All [Special Tags](../../special-tags) apply as well.
```
Comment (bsmid) [inline docs/logs/special-fields.md]: Maybe reword to something like "Common Special Tags apply to Sematext Logs. Additionally, there are Logs-specific fields which also get special treatment:.."

File: docs/monitoring/tag-support.md [hunk 2]
```diff
@@ -2,109 +2,12 @@ title: Monitoring Tags
...
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
 
-| Tag Name  | Description  | Synonymous Tags
-|:--|:--|:--
-| os.host.ip | IP Address of the host/server | server.ip, server.address, host.ip, host.address, source.ip, source.address
-| service.name | Name of the service where the data is collected from |
-| service.id | Unique service identifier |
-| service.type | Service type e.g. `hadoop` |
-| span | Building block of trace in distributed tracing |
+All [Special Tags](../../special-tags) apply as well.
```
Comment (bsmid) [inline docs/monitoring/tag-support.md]: Maybe change to "At the moment all Common Special Tags apply to monitoring Apps".

File: docs/monitoring/tag-support.md [hunk 3]
```diff
@@ -2,109 +2,12 @@ title: Monitoring Tags
...
 
+The tags are automatically collected by agents while fetching metrics and sent as part of metrics to the receiver.
```
Comment (bsmid) [inline docs/monitoring/tag-support.md]: I think not needed as we say that in line 4. We could mention that in case when 3rd party Agents are used, they should ensure tag names match those from our Common Special Tags.

File: docs/special-tags.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+title: Special Tags 
+description: The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. They are used to correlate data across multiple Apps.
+
+Tags are sent by Sematext Agents as part of every data point and they are shown in UI as filters.
```
Comment (bsmid) [inline docs/special-tags.md]: "... every data point or log line..."

File: docs/special-tags.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
+title: Special Tags 
+description: The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. They are used to correlate data across multiple Apps.
+
+Tags are sent by Sematext Agents as part of every data point and they are shown in UI as filters. 
+
+The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. They are used to correlate data across multiple Apps. Refer to [Special Logs Fields](./logs/special-fields/) for list of special fields in logs.
```
Comment (bsmid) [inline docs/special-tags.md]: "... for list of additional special fields for logs. Monitoring Apps currently use only common special tags described on this page."

File: docs/special-tags.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+
+## Common Tags
+
+The tags below are applicable to all metrics types:
```
Comment (bsmid) [inline docs/special-tags.md]: I guess wherever we show "metrics" on this page, we should adjust to "metrics/logs".

File: docs/monitoring/tag-support.md (hunk 3 above)
Comment (adnanrahic) [reply docs/monitoring/tag-support.md]: I'd just remove this altogether. Also, I'd not mention 3rd party Agents. I want to push only our Agents in our Docs.

Comment (bsmid) [reply docs/monitoring/tag-support.md]: Ability to use 3rd party agents is an important selling point.

Comment (adnanrahic) [reply docs/monitoring/tag-support.md]: Aha, okay. I'll add a mention of this.
